### PR TITLE
redesign to reflect VegaLite examples and structure

### PR DESCRIPTION
**Pull request by oheil**
moved and added examples.
Still missing some examples, e.g. the new Advanced Calculations.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ JSONSchema = "7d188eb4-7ad8-530c-ae41-71a32a6d4692"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Query = "1a8c2f83-1ff3-5112-b086-8aa67b057ba1"
```
Comment (davidanthoff) [inline Project.toml]: I think this can/should go into `docs/Project.toml`, right?

Comment (oheil) [reply Project.toml]: It is already there.
I think it has been added automatically to the root Project.toml somehow and I ignored it.

**Comment by davidanthoff** [comment]
Do you want me to merge this now and do the next steps as follow up PRs, or should I wait? My preference would be to merge this, and then do more PRs.

**Comment by oheil** [comment]
Yes, my intention was, this PR as a kind of question if this should be the future structure. If the PR is merged I am happy to proceed. So, yes, I want it to be merged :-)

**Comment by oheil** [comment]
Yes, my intention was, this PR as a kind of question if this should be the future structure. If the PR is merged I am happy to proceed. So, yes, I want it to be merged :-)

**Comment by davidanthoff** [comment]
Done :)

**Pull request merged**